content/ru/docs/concepts/workloads/pods/_index.md

```diff
@@ -0,0 +1,395 @@
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}} или 
+{{< glossary_tooltip text="Job" term_id="job" >}}. Если вашим Подам необходимо отслеживать состояние, 
+рассмотрите ресурс {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}.
+
+
+Каждый Под предназначен для запуска одного экземпляра приложения. Если вы хотите горизонтально 
+масштабировать приложение (обеспечить больше общих ресурсов за счет запуска большего количества экземпляров), 
+вам следует использовать несколько Подов, по одному для каждого экземпляра. В Kubernetes это обычно называется 
+_репликацией_. Реплицированные Поды обычно создаются и управляются как группа ресурсом рабочей нагрузки и его 
+{{< glossary_tooltip text="контроллером" term_id="controller" >}}.
+
+Подробнее о том, как Kubernetes использует ресурсы рабочей нагрузки и их контроллеры для масштабирования и 
+автовосстановления приложений, читайте в разделе [Поды и контроллеры](#pods-and-controllers).
+
+Поды изначально предоставляют два вида общих ресурсов для входящих в них контейнеров: [сеть](#pod-networking) 
+и [хранилище](#pod-storage).
+
+## Работа с Подами {#working-with-pods}
+
+Вы редко будете создавать отдельные Поды непосредственно в Kubernetes-даже Поды-одиночки. Это связано 
+с тем, что Поды разработаны как относительно эфемерные, одноразовые сущности. Когда Под создается 
+(непосредственно вами или косвенно {{< glossary_tooltip text="контроллером" term_id="controller" >}}), 
+новый Под планируется к запуску на {{< glossary_tooltip text="узле" term_id="node" >}} в вашем кластере. 
+Под остается на этом узле до тех пор, пока Под не завершит выполнение, объект Пода не будет удален, 
+Под не будет вытеснен из-за нехватки ресурсов, или узел не выйдет из строя.
+
+{{< note >}}
+Не следует путать перезапуск контейнера в Поде с перезапуском Пода. Под - это не процесс, а среда для 
+запуска контейнера(ов). Под сохраняется до тех пор, пока не будет удален.
+{{< /note >}}
+
+Имя Пода должно быть действительным значением 
+[имени поддомена DNS](/ru/docs/concepts/overview/working-with-objects/names#dns-subdomain-names), но это 
+может привести к неожиданным результатам для имени хоста Пода. Для лучшей совместимости имя должно 
+соответствовать более строгим правилам для 
+[имен меток DNS](/ru/docs/concepts/overview/working-with-objects/names#dns-label-names).
+
+### ОС Пода {#pod-os}
+
+{{< feature-state state="stable" for_k8s_version="v1.25" >}}
+
+В поле `.spec.os.name` следует установить значение `windows` или `linux`, чтобы указать ОС, на которой 
+будет работать под. На данный момент Kubernetes поддерживает только эти две операционные системы. 
+В будущем этот список может быть расширен.
+
+В Kubernetes v{{< skew currentVersion >}} значение, заданное для этого поля, не влияет на 
+{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}} данных подов. 
+Задание `.spec.os.name` помогает авторитетно идентифицировать ОС пода и используется для проверки. 
```

Comment:
This and the following sentence seem to be outdated compared with the [current English text](https://kubernetes.io/docs/concepts/workloads/pods/#pod-os).